src/kaleidoscope/device/dygma/raise/SideFlash.h

```diff
@@ -20,7 +20,7 @@
 
 #ifdef ARDUINO_SAMD_RAISE
 
-#include <Kaleidoscope.h>
+#include <Runtime.h>
```

Comment:
Should all of these "#include <Runtime.h>" be "#include "kaleidoscope/Runtime.h" instead?

Reply:
Correct. The Raise was the only device that got not updated and still used the old service-include file.

Will fix that.